pkg/sources/github/github.go

```diff
@@ -678,6 +693,7 @@ func (s *Source) scan(ctx context.Context, installationClient *github.Client, ch
 			path, repo, err = s.cloneRepo(ctx, repoURL, installationClient)
 			if err != nil {
 				scanErrs.Add(err)
+				return nil
```

Comment:
is this related to the data race?

Reply:
This was causing a test to fail. If we can't clone a repo we should not try to scan.